buildout.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://zopefoundation.github.io/Zope/releases/4.x/versions.cfg
+    http://zopefoundation.github.io/Zope/releases/master/versions.cfg
```

Comment:
Is the usage of http instead of https intended?

Reply:
Good catch of a copy-paste-mistake.